core/domain/exp_services_test.py

```diff
@@ -924,6 +924,17 @@ def setUp(self):
             'param_changes': []
         }
 
+    def test_add_state_name(self):
+        """Test adding of state"""
+        exploration = exp_services.get_exploration_by_id(self.EXP_ID)
+        exp_services.update_exploration(self.owner_id, self.EXP_ID, [{
+            'cmd': 'add_state',
```

Comment:
Instead of ``'add_state'``, can you update this to be ``exp_domain.CMD_ADD_STATE``? Also, can you update all other calls to update_exploration() in this test to use constants from ``exp_domain`` rather than direct strings?

Once done, can you make sure all of the commands in ``exp_domain`` are being tested via ``update_exploration()`` calls throughout this test suite?